addons/account/views/account_move_views.xml

```diff
@@ -162,6 +162,7 @@
             <field eval="100" name="priority"/>
             <field name="arch" type="xml">
                 <tree string="Journal Items" create="false" edit="true" expand="context.get('expand', False)" multi_edit="1" sample="1">
+                    <field name="move_id" readonly="1"/>
```

Comment:
Do we really want to display it in the view? Maybe consult with a PO to ensure if this field should be visible or invisible? I would go for invisible personally.

Reply:
nope, I didn't look again it was supposed to be invisible I think I'll change and try again

Reply:
Still works with invisible, nice catch :+1: